src/Forms/GridField/GridFieldPrintButton.php

```diff
@@ -258,6 +267,8 @@ public function generatePrintData(GridField $gridField)
             }
         }
 
+        $gridFieldColumnsComponent?->setEscapeFields($origEscape);
```

Comment:
```suggestion
        $gridFieldColumnsComponent?->setDoEscapeFields($origDoEscapeFields);
```

Reply:
I don't think the parameter has to match the exact same name as the property that's being set, and I'd personally prefer to omit `Fields` from it - but it's very subjective and super doesn't matter so I'll make the change

Reply:
Done